test/common/transports.test.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-env jest */
+
+const logdown = require('../../src/node')
+
+describe('logdown.transports', () => {
+  let initialDebug
+
+  beforeEach(() => {
+    logdown.transports.length = 0 // empty an array
```

Comment:
Why not `transports = []`?

Reply:
actually, that was for fun :D
I'll change back to `[]`